### ISSUE TITLE
Shiny

### ISSUE THREAD

**Issue by josep2** [hrbrmstr/streamgraph#6]
Does this library currently support shiny?

**Comment by hrbrmstr** [comment]
Haven't tried I but it should. I'll whip up an example later tonight to test

**Comment by hrbrmstr** [comment]
Yep. It seems to do just fine.

    library(streamgraph)
    library(dplyr)
    library(shiny)

    ui = shinyUI(fluidPage(
      h3("streamgraph example", style="text-align:center"),
      streamgraphOutput('sg1')
    ))

    server = function(input, output) {
      
      ggplot2::movies %>%
        select(year, Action, Animation, Comedy, Drama, Documentary, Romance, Short) %>%
        tidyr::gather(genre, value, -year) %>%
        group_by(year, genre) %>%
        tally(wt=value) %>%
        ungroup %>%
        streamgraph("genre", "n", "year") %>%
        sg_axis_x(20) %>%
        sg_fill_brewer("PuOr") %>%
        sg_legend(show=TRUE, label="Genres: ") -> sg
      
      output$sg1 <- renderStreamgraph(sg)

    }

    shinyApp(ui = ui, server = server)

**Comment by flopesdematos** [comment]
I have a interactive stream graph working perfectly with my data on a Shiny app following this example. However, when I tried to put the same graph on a absolutePanel similar to the one on the interactive map of the [SuperZip](https://github.com/rstudio/shiny-examples/tree/master/063-superzip-example) example, the values associated to each graph band through the X axis are incorrect. I tried different panel widths but the problem persists, although with different erroneous values according to the date. Is it possible to keep the absolutePanel with a reasonable width and the stream graph working properly? 

Thank you!
